src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -54,23 +54,16 @@ public ExploreService(
         this.caseService = caseService;
     }
 
-    public void createStudy(String studyName, UUID caseUuid, String description, String userId, UUID parentDirectoryUuid, Map<String, Object> importParams) {
+    public void createStudy(String studyName, UUID caseUuid, String description, String userId, UUID parentDirectoryUuid, Map<String, Object> importParams, Boolean duplicateCase) {
```

Comment:
Use primitive type

Reply:
Was done in commit https://github.com/gridsuite/explore-server/pull/42/commits/d16af1720af9c107ce5efbed7adccf1f634c4bd0